talk/concurrency/mutexes.tex

```diff
@@ -184,10 +185,10 @@
       \begin{itemize}
       \item Or add master lock protecting the locking phase
       \end{itemize}
-    \item Respect a strict order in the locking across all threads
+    \item Respect a strict locking order across all threads
```

Comment:
You could mention that `std::scoped_lock` enforces uses a dead-lock avoidance algorithm if used on multiple mutexes.

Reply:
You mean like 5 lines above?
> `\item \cpp17: \mintinline{cpp}{std::scoped_lock lock{m1, m2};} comes with deadlock-avoidance algorithm`

Yes, I agree! 😂

Reply:
Yeah!